src/benchmark_environments/util.py

```diff
@@ -17,3 +20,107 @@ def step(self, action):
             info["terminal_observation"] = obs
             obs = self.env.reset()
         return obs, rew, False, info
+
+
+class EpisodeEndRewardWrapper(gym.Wrapper):
+    """Replaces all rewards, with all rewards becoming zero except episode end.
+
+    Useful for converting living rewards into equivalent episode termination
```

Comment:
Can you make explicit the episode terminates when some special condition is reached, and not (in its intended use case) when it timesout?

If episode termination is *always* reached, then in the undiscounted case all policies are optimal, and even in the discounted case the incentives are quite weak.

Reply:
I removed this wrapper (because creates non-reward equivalent variants of gym envs) so this comment is now n/a